sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/Configs.java

```diff
@@ -44,6 +47,7 @@ public class Configs {
 
     private static final String UNAVAILABLE_LOCATIONS_EXPIRATION_TIME_IN_SECONDS = "COSMOS.UNAVAILABLE_LOCATIONS_EXPIRATION_TIME_IN_SECONDS";
     private static final String GLOBAL_ENDPOINT_MANAGER_INITIALIZATION_TIME_IN_SECONDS = "COSMOS.GLOBAL_ENDPOINT_MANAGER_MAX_INIT_TIME_IN_SECONDS";
+    private static final String THINCLIENT_ENDPOINT = "COSMOS.THINCLIENT_ENDPOINT"; // Environment variable for now
```

Comment:
This is also where you need to add a sytsem-property/env varaiable based way to tell whethe rthinclient is enabled (during bake time should defualt to false, later we will just enable it )

Reply:
I thought we had previously discussed using one of the http2 flags as a switch for enabling thin client - I am ok to make this change but just want to get consensus on this taking this approach